CMakeLists.txt

```diff
@@ -70,11 +72,11 @@ INCLUDE(FindPkgConfig)
 PKG_SEARCH_MODULE(FT2 REQUIRED freetype2)
 PKG_SEARCH_MODULE(SDL2 REQUIRED sdl2)
 PKG_SEARCH_MODULE(SDL2IMAGE REQUIRED SDL2_image>=2.0.0)
-PKG_SEARCH_MODULE(GLU REQUIRED glu)
+# PKG_SEARCH_MODULE(GLU REQUIRED glu)
```

Comment:
I removed the the package search for GLU since it didn't seem to be needed on linux. But maybe I'm wrong 🤔

Reply:
indeed